docs/develop/mainnet/auditing.md

```diff
@@ -0,0 +1,24 @@
+# Security Audits
```

Comment:
Consider moving this to testnet, as audits should be performed before moving to mainnet

Reply:
moved